modules/EnsEMBL/Web/Query/Generic/Base.pm

```diff
@@ -91,7 +91,7 @@ sub loop_species {
   my ($self,$args,$subpart) = @_;
 
   my @out;
-  foreach my $sp (qw(Homo_sapiens Mus_musculus Danio_rerio)) {
+  foreach my $sp (@{$SiteDefs::PRECACHE_DEFAULT_SPECIES}) {
```

Comment:
`$SiteDefs::PRECACHE_DEFAULT_SPECIES` is set only in the precachefarm plugins. 
So if the website uses this module there isn't a default value?

Reply:
Hi @tuanebi  This is only used by precache (checked code, also confirmed by @ens-ds23) for internal use. So do we need to a default value added to common Sitedefs?

Reply:
Only precache uses it so adding to common is unnecessary I think